docs/topics/commonly_used_plugins.rst

```diff
@@ -10,284 +10,78 @@ Some commonly-used plugins
-running.
-
-.. note:: For more advanced use cases where you would like to upload your media
-          to a central location, consider using  `django-filer`_ with
-          `django filer CMS plugin`_ and its ``cmsplugin_filer_video`` component
-          instead.
-
-.. _django-filer: https://github.com/stefanfoulis/django-filer
-.. _django filer CMS plugin: https://github.com/stefanfoulis/cmsplugin-filer
-
-..  module:: djangocms_twitter
-
-..  class:: djangocms_twitter.cms_plugins.TwitterRecentEntriesPlugin
-
-..  class:: djangocms_twitter.cms_plugins.TwitterSearchPlugin
-
-*******
-Twitter
-*******
-
-We recommend one of the following plugins:
-
-* https://github.com/nephila/djangocms_twitter
-* https://github.com/changer/cmsplugin-twitter
-
-.. warning:: These plugins are not currently compatible with Django 1.7.
-
-..  module:: djangocms_inherit
-
-..  class:: djangocms_inherit.cms_plugins.InheritPagePlaceholderPlugin
-
-*******
-Inherit
-*******
-
-Available on `GitHub (divio/djangocms-inherit) <http://github.com/divio/djangocms-inherit>`_
-and on `PyPi (djangocms-inherit) <https://pypi.python.org/pypi/djangocms-inherit>`_.
-
-Displays all plugins of another page or another language. Great if you always
-need the same plugins on a lot of pages.
-
-Please install it using ``pip`` or similar and be sure you have the following in your project's
-:setting:`django:INSTALLED_APPS` setting::
-
-    INSTALLED_APPS = (
-        # ...
-        'djangocms_inherit',
-        # ...
-    )
-
-.. warning:: The inherit plugin **cannot** be used in non-cms placeholders.
-
-.. _Django Packages: http://djangopackages.com/grids/g/thumbnails/
-.. _easy-thumbnails: https://github.com/SmileyChris/easy-thumbnails
+Please note that dozens if not hundreds of different django CMS plugins have been made available
+under open-source licences. Some, like the ones on this page, are likely to be of general interest,
+while others are highly specialised.
+
+This page only lists those that fall under the responsibility of the django CMS project. Please see
+the `Django Packages <https://djangopackages.org/search/?q=django+cms>`_ site for some more, or
+just do a web search for the functionality you seek - you'll be surprised at the range of plugins
+that has been created.
+
+
+**********************
+django CMS Core Addons
+**********************
+
+We maintain a set of *Core Addons* for django CMS. 
+
+You don't need to use them, and for many of them alternatives exist, but they represent a good way
+to get started with a reliable project set-up. We recommend them for new users of django CMS in
+particular. For example, if you start a project on `Divio Cloud <https://divio.com/>`_ or using the
+`django CMS installer <https://github.com/nephila/djangocms-installer>`_, this is the set of addons
+you'll have installed by default.
+
+The django CMS Core Addons are:
+
+* `django Filer <http://github.com/divio/djangocms-file>`_ - a file management application for
```

Comment:
please consider the ordering https://github.com/divio/django-cms/pull/5942/#discussion_r113393518